### ISSUE TITLE
legend support with GetLegendGraphic

### ISSUE THREAD

**Issue by sheppard** [heigeo/leaflet.wms#4]
Integrate legend support (edit: using `GetLegendGraphic`).  The main question is where to put the legend - probably in a custom control that defaults to the bottom left.

**Comment by vlasvlasvlas** [comment]
how about using directly WMS GetLegendGraphic?

```
url: http://mapserver.org/ogc/wms_server.html
```

GetLegendGraphic: returns a legend image (icon) for the requested layer, with label(s). 

This request returns a legend image (icon) for the specified layer. The request will draw an icon and a label for all classes defined on the layer. If the requested layername is a GROUP-name, all included layers will be returned in the legend-icon.

**Comment by sheppard** [comment]
Yes, `GetLegendGraphic` would be the best approach.  It shouldn't be too hard to generate an appropriate URL based off of the base WMS url.
